intermediate-solution/index.html

```diff
@@ -0,0 +1,500 @@
+
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+    <script async custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>
+    <script async custom-element="amp-twitter" src="https://cdn.ampproject.org/v0/amp-twitter-0.1.js"></script>
+    <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+    <script async custom-element="amp-sidebar" src="https://cdn.ampproject.org/v0/amp-sidebar-0.1.js"></script>
+
+    <script custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js" async=""></script>
+    <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-0.1.js"></script>
+    <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+    <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+    <script async custom-element="amp-lightbox" src="https://cdn.ampproject.org/v0/amp-lightbox-0.1.js"></script>
+    <script async custom-element="amp-selector" src="https://cdn.ampproject.org/v0/amp-selector-0.1.js"></script>
+    <script custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js" async></script>
+
+    <style amp-custom>
+      body {
+        font-family: sans-serif;
+        line-height: 1.5rem;
+        min-width: 315px;
+      }
+
+      .headerbar {
+        align-items: center;
+        background-color: #fff;
+        box-shadow: 0 0 5px 2px rgba(0,0,0,.1);
+        display: flex;
+        left: 0;
+        padding-left: 1rem;
+        position: fixed;
+        right: 0;
+        top: 0;
+        z-index: 999;
+      }
+
+      .headerbar+:not(amp-sidebar),.headerbar+amp-sidebar+* {
+        margin-top: 3.5rem;
+      }
+
+      h1,h2,h3,h4,h5,h6 {
+        font-weight: 400;
+        letter-spacing: .06rem;
+        margin: 0;
+        padding: 0;
+      }
+
+      .h2,h2 {
+        font-size: 2rem;
+        line-height: 2.5rem;
+      }
+
+      .h3,h3 {
+        font-size: 1.5rem;
+        line-height: 2rem;
+      }
+
+      .sidebar .dropdown-item,.sidebar .dropdown header,.sidebar .nav-item {
+        margin: 0 0 2rem;
+      }
```

Comment:
Can you do something to make it clear that these are submenu items?  Indent, change color a bit, etc?

I started with the same sample code [here](http://delsolquartet.com/angel-island-live.html) and kind of like how it turned out...